client/plots/violin.interactivity.js

```diff
@@ -91,17 +88,16 @@ export function setInteractivity(self) {
 		}
 		plot.start = start
 		plot.end = end
-		self.displayMenu(event, options, plot)
+		self.displayMenu(event, options, plot, true)
```

Comment:
since start/stop is only generated by brushing and not present when not brushing, just pass the optional start/stop to function, instead of the flag. and not a good idea to assign transient start/stop on plot

Reply:
okay I just made that fix. Thanks for the pointer